go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/troubleshooting-panel-console-plugin
 
-go 1.22.5
+go 1.22
```

Comment:
The go version needs to stay as x.y.z for the appropriate toolchain version to be downloaded for cachito based builds ([thread](https://redhat-internal.slack.com/archives/C02AX10EQJW/p1723813393142139?thread_ts=1723737266.170519&cid=C02AX10EQJW)).

Reply:
The dockerfile appears to use a 1.22.1 image, so I guess that's the right value for now...